tests/chains/Elrond/TWCoinTypeTests.cpp

```diff
@@ -27,8 +27,8 @@ TEST(TWElrondCoinType, TWCoinType) {
     ASSERT_EQ(0x0, TWCoinTypeP2shPrefix(TWCoinTypeElrond));
     ASSERT_EQ(0x0, TWCoinTypeStaticPrefix(TWCoinTypeElrond));
     assertStringsEqual(symbol, "eGLD");
-    assertStringsEqual(txUrl, "https://explorer.elrond.com/transactions/1fc9785cb8bea0129a16cf7bddc97630c176a556ea566f0e72923c882b5cb3c8");
-    assertStringsEqual(accUrl, "https://explorer.elrond.com/address/erd12yne790km8ezwetkz7m3hmqy9utdc6vdkgsunfzpwguec6v04p2qtk9uqj");
+    assertStringsEqual(txUrl, "https://explorer.multiversx.com/transactions/1fc9785cb8bea0129a16cf7bddc97630c176a556ea566f0e72923c882b5cb3c8");
```

Comment:
This tx explorer link gives Not found (same with old elrond explorer), it should be changed to a valid TX ID.

Reply:
Indeed. Fixed: added a working example (also updated the _accounts_ URL).